device-types/SMC Networks/SMCGS8P-SMART.yaml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: SMC Networks
+model: SMCGS8P-SMART
+slug: smc-networks-smcgs8p-smart
+part_number: SMCGS8P-SMART
+comments: '[Datasheet](https://www.use-ip.co.uk/datasheets/442/smc_gs8p_8port_gigabit_poe_switch.pdf)'
+airflow: left-to-right
+weight: 2.06
+weight_unit: kg
+is_full_depth: false
+front_image: true
+rear_image: true
+u_height: 1
+interfaces:
+  - name: GigabitEthernet1
+    type: 1000base-t
```

Comment:
Looking at the spec sheet these are poe interfaces therefore they need the following poe config for each interface
'poe_mode: pse'
'poe_type: type1-ieee802.3af'